From 3385dfd30c58a9d8c1de0f4e99d7a4b4a54abca7 Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@liri.io>
Date: Sat, 13 May 2017 22:30:59 +0200
Subject: [PATCH 2/2] Fix build with OpenEmbedded

Exclude tests from build since they require gui which is not available
for -native builds and remove conflict with cross-compilers.
---
 qbs.pro | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/qbs.pro b/qbs.pro
index af6ba896..5d964645 100644
--- a/qbs.pro
+++ b/qbs.pro
@@ -1,5 +1,3 @@
-requires(!cross_compile)
-
 defineTest(minQtVersion) {
     maj = $$1
     min = $$2
@@ -43,8 +41,7 @@ SUBDIRS += \
     src_app\
     src_libexec\
     src_plugins\
-    static.pro\
-    tests
+    static.pro
 
 OTHER_FILES += \
     doc/*.qdoc \
-- 
2.12.2

